### DISCUSSION TITLE
Using Firebase server values

### DISCUSSION THREAD

**Discussion by george-kar** [SchneiderInfosystems/FB4D#154] [Q&A] [answered]
According to Firebase Rest SDK there are two server variables **timestamp** and **increment**:

https://firebase.google.com/docs/reference/rest/database#section-server-values

is there any way to use them on the delphi implementation?

**Comment by SchneiderInfosystems** [comment, marked as answer]
Please read firstly the Wiki or order your privat learning session:

https://github.com/SchneiderInfosystems/FB4D/wiki/FB4D-Reference-IRealTimeDB#get-a-server-variable
